spec/helpers/navbar_helper_spec.rb

```diff
@@ -26,10 +26,30 @@
         expect(fragment).to have_css 'nav.bg-purple-700.py-4.px-8.flex.justify-between.items-center'
       end
 
-      it 'renders navbar with title' do
-        fragment = view.tramway_navbar(title:)
+      context 'with title checks' do
+        it 'renders navbar with title' do
+          fragment = view.tramway_navbar(title:)
 
-        expect(fragment).to have_content title
+          expect(fragment).to have_content title
+        end
+
+        it 'renders navbar with default title link' do
+          fragment = view.tramway_navbar(title:)
+
+          expect(fragment).to have_css "a[href='/']"
+        end
```

Comment:
Looks like the line `33` and `39` can be in the same test.

```rb
describe 'default title link' do
  let(:fragment) { view.tramway_navbar(title:) }
  
  it 'renders navbar with title' do
    expect(fragment).to have_content(title:)
  end

  it 'renders navbar with default title link' do
    expect(fragment).to have_css "a[href='/']"
  end
end
```

or we can do this:

```rb
describe 'default title link' do
  subject { view.tramway_navbar(title:) }
  
  it { is_expected.to have_content(title:) }
  it { is_expected.to have_css "a[href='/']" }
end
```

Reply:
Merged! Thanks!